lib/rollbar/scrubbers/params.rb

```diff
@@ -56,6 +57,9 @@ def build_whitelist_regex(whitelist)
       end
 
       def scrub(params, options)
+        return params if @scrubbed_object_ids[params.object_id].present?
```

Comment:
`#present?` isn't guaranteed to be available in non-Rails host apps. (Rack, Sinatra) 
```
return params if @scrubbed_object_ids[params.object_id]
```
is best here.